lib/utils/network.dart

```diff
@@ -56,7 +56,7 @@ class NetworkUtil {
     }
 
     try {
-      var stream = new http.ByteStream(DelegatingStream.typed(img.openRead()));
+      var stream = new http.ByteStream(stream.cast(img.openRead()));
```

Comment:
1. It should be `Stream.castFrom`
2. We have already declared the `stream` so we need to change the variable name as well so,
```diff
- var stream = new http.ByteStream(DelegatingStream.typed(img.openRead()));
+ var streamed = new http.ByteStream(Stream.castFrom(img.openRead()));
.
.
.
- var multipartFile = new http.MultipartFile('image', stream, length,
+ var multipartFile = new http.MultipartFile('image', streamed, length,
```